spark/components/buttons/vanilla/button.stories.js

```diff
@@ -1,48 +1,69 @@
-import { withKnobs, text } from '@storybook/addon-knobs';
 
 export default {
   title: 'Components|Buttons',
-  decorators: [withKnobs],
 };
 
 export const primary = () => (
   `
     <button class="sprk-c-Button" data-id="button-1">
-      ${text('button text', 'Button')}
+      Button
     </button>
   `
 );
 
 export const secondary = () => (
-  '<button class="sprk-c-Button sprk-c-Button--secondary" type="button" data-id="button-1">Button</button>'
+  `
+    <button class="sprk-c-Button sprk-c-Button--secondary" type="button" data-id="button-1">
+    Button
+    </button>
+  `
 );
 
 export const tertiary = () => (
-  '<button class="sprk-c-Button sprk-c-Button--tertiary" data-id="button-1">Button</button>'
+  `
+    <button class="sprk-c-Button sprk-c-Button--tertiary" type="button" data-id="button-2">
+    Button
+    </button>
+  `
 );
 
 export const disabled = () => (
-  '<button class="sprk-c-Button sprk-is-Disabled" data-id="button-1" disabled>Button</button>'
+  `
+  <button class="sprk-c-Button sprk-is-Disabled" data-id="button-1" disabled>
+    Button
+  </button>
+  `
 );
 
-const withSpinner = () => (
-  '<button class="sprk-c-Button" data-sprk-spinner="click" data-id="button-spinner-1">Submit</button>'
+export const spinner = () => (
+  `
+  <button class="sprk-c-Button" data-sprk-spinner="click" data-id="button-spinner-1">
+    <div class="sprk-c-Spinner sprk-c-Spinner--circle"></div>
+  </button>
+  `
 );
-
-withSpinner.story = {
-  name: 'with spinner',
+spinner.story = {
+  name: 'Spinner',
 };
 
-export const fullWidthAtSmallViewport = () =>
-  '<button class="sprk-c-Button sprk-c-Button--full@s" type="button" data-id="button-1">Button</button>';
-
+export const fullWidthAtSmallViewport = () => (
+  `
+  <button class="sprk-c-Button sprk-c-Button--full@s" type="button" data-id="button-1">
```

Comment:
@RVMendoza , this one has the same data-id as the secondary and disabled buttons and needs to be unique.